### ISSUE TITLE
Locking migration generation to specified tables

### ISSUE THREAD

**Issue by killua-eu** [odan/phinx-migrations-generator#91]
When designing a new database layout, I often get myself into a situation, where a part of changes made to tables is already done (ready for a commit), whereas other tables are still being a wip. Is it possible to restrict the migration generation only to tables specified? I'm looking at how

```
git add * && git commit -m "some work"
./vendor/bin/phinx-migrations generate --name 'some work' --overwrite
```

could compare compare to, if a parameter such as `--tables` would exist

```
git add x1 x2 && git commit -m "some work"
./vendor/bin/phinx-migrations generate --name 'some work' --overwrite --tables x1 x2
```

is this an existing feature?

**Comment by odan** [comment]
This feature does not exist. 

Usually I do this by generating a smaller migration when I know that the change is done. 
Then I change the next table(s) and generate a new migration for it.

**Comment by killua-eu** [comment]
I tried and failed miserably (to generate smaller migrations). I'm too unstructured when updating databases and have a tendency to work on several things in parallel. This would definitely help my more chaotic workflow.
